Doc/library/typing.rst

```diff
@@ -1372,3 +1388,87 @@ The module defines the following classes, functions and decorators:
    evaluated, so the second annotation does not need to be enclosed in quotes.
 
    .. versionadded:: 3.5.2
+
+.. data:: Annotated
```

Comment:
Can I ask that you hint at the possibility of passing multiple extras (`Annotated[T, x, y, z]`) early in the description?